broker/lead.go

```diff
@@ -115,6 +109,41 @@ func lead(authToken models.AuthToken, policy *models.Policy) error {
 	return err
 }
 
+func checkIfPolicyIsLead(policy *models.Policy) error {
+	var recoveredPolicy models.Policy
+	log.Printf("[lead] found lead for existing policy %s", policy.Uid)
```

Comment:
move log to line 129, after we successfully retrieved the policy from the DB

Reply:
Done in https://github.com/wopta/goworkspace/pull/121/commits/0d1cfbfbdb3a1bfc445c793fda5ffeef4f83cc4c